lorien/InfiniteCanvas/Cursor/BaseCursor.gd

```diff
@@ -5,10 +5,21 @@ extends Sprite
 var _brush_size: int
 var _pressure := 1.0
 
+onready var _camera: Camera2D = get_viewport().get_node("Camera2D")
```

Comment:
Self-critique: This is "not-so-nice". I want to find the active Camera2D, but godot Viewport's function get_camera() only returns the current 3D camera! So there is no easy way to find it. I guess this is good enough and clear enough to leave it for now (crashes when someone should move/rename the camera). But if you really want me to, I can refactor this to find the proper active camera of the viewport...

Reply:
I have done something like this a couple of times myself - not proud, but not a big deal if used sparingly imo.